src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -692,5 +692,153 @@ public static void SyndicationFeed_Rss_WrongDateFormat()
             // *** ASSERT *** \\
             Assert.True(!res.LastUpdatedTime.Equals(new DateTimeOffset()));
         }
+        
+        [Fact]
+        [ActiveIssue(24571)]
+        public static void AtomEntryPositiveTest()
+        {
+            string filePath = @"brief-entry-noerror.xml";
+            string serializeFilePath = Path.GetTempFileName();
+            
+            try
+            {
+                XmlReader reader = XmlReader.Create(filePath , new XmlReaderSettings() {Async=true});
+                Task<SyndicationItem> task = SyndicationItem.LoadAsync(reader);
+                Task.WhenAll(task);
```

Comment:
This is a nop.  WhenAll creates a task that will complete when all of the tasks passed to it have completed.  Here you're passing it just one task, so the WhenAll isn't necessary, and you're not awaiting the result of the WhenAll, so nothing is waiting for anything.  This should just be:
```C#
SyndicationItem feedObject = await SyndicationItem.LoadAsync(reader);
```

Reply:
Thanks, I have changed the code as you suggested.